base/settings/R/check.all.settings.R

```diff
@@ -398,7 +398,7 @@ check.settings <- function(settings, force = FALSE) {
         "qsub not specified using default value :", settings$host$qsub)
     }
     if (is.null(settings$host$qsub.jobid)) {
-      settings$host$qsub.jobid <- "Your job ([0-9]+) .*"
+      settings$host$qsub.jobid <- "Your job ^[A-Za-z0-9_ -]*$"
```

Comment:
```suggestion
      settings$host$qsub.jobid <- "[^0-9]*([0-9]+)[^0-9]*"
```

Reply:
I'm going to go ahead and commit this suggestion since there hasn't been any activity on this PR for a while